docs/unit.md

```diff
@@ -41,10 +88,5 @@ aw-browser-test-runner ./test/unit
   * `-d`|`--dirs`: Paths to directories to serve.
   * `--phantomjs-single-run`: Run once.
 
-## Debugging
-
-### Node context
-Add the `--debug` argument to the aw-test-runner CLI command. The script will break on the very first line in **Chrome DevTools**.
-
-### Browser context
+#### Debugging Browser context
 By default, this driver will run the tests inside Chrome. You can bring up **Chrome DevTools**, set the break points, and re-run the tests after you have run the tests once.
```

Comment:
This driver runs the tests inside Chrome by default. Bring up **Chrome DevTools*, set the break points, and re-run the tests after you have run the tests once.